addon/components/head-layout.js

```diff
@@ -1,37 +1,30 @@
-import Component from '@ember/component';
+import Component from '@glimmer/component';
 import { inject as service } from '@ember/service';
-import layout from '../templates/components/head-layout';
```

Comment:
Let's either use colocated template + component, or bring this import back and use `setComponentTemplate` directly (which is the analogue to setting `layout` property on the prototype).

The second option might look like:

```js
import { setComponentTemplate } from '@ember/component';
import layout from '../templates/components/head-layout';

export default class HeadLayout extends Component {
  // ..snip...
}

setComponentTemplate(layout, HeadLayout);
```

---

Just to note: this is basically how colocated templates work.